src/ldc/eh/common.d

```diff
@@ -300,6 +360,11 @@ struct ActiveCleanupBlock {
     /// The exception that caused this cleanup block to be entered.
     Object dObject;
 
+    /// The exception struct associated with the above exception. Currently only
+    /// used by libunwind; define _d_eh_destroy_exception_struct in implementation-specific
+    /// code to handle this correctly.
+    void* exceptionStruct;
```

Comment:
Indeed; I investigated replacing `dObject` with the exception struct, but that poses issues with Win64. This was the cleanest approach that I could see.